lib/appsignal/diagnose/library.ex

```diff
@@ -4,7 +4,7 @@ defmodule Appsignal.Diagnose.Library do
   require Appsignal.Utils
 
   @appsignal_version Mix.Project.config()[:version]
-  @agent_version Appsignal.Nif.agent_version()
```

Comment:
Can we also remove this function, or is it used anywhere else?

Reply:
Yes, I think we can remove it.